sdk/core/azure-core/src/main/java/com/azure/core/util/serializer/jsonwrapper/api/Type.java

```diff
@@ -1,9 +1,13 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Same here.

Reply:
Type is being implemeted in jacksonwrapper.JacksonNode and in the unit tests